analyseCI.sh

```diff
@@ -1,29 +1,22 @@
 #!/bin/bash
 
-# This script will be executed by the CI software (Usually Jenkins)
-# This script CAN'T have any root access.
-# All information print on stdout will be printed by the CI software.
-
-#=================================================
-# Grab the script directory
-#=================================================
-
 if [ "${0:0:1}" == "/" ]; then script_dir="$(dirname "$0")"; else script_dir="$(echo $PWD/$(dirname "$0" | cut -d '.' -f2) | sed 's@/$@@')"; fi
 
-#=================================================
-# Check the number of arguments
-#=================================================
+cd $script_dir
 
 if [ $# -ne 2 ]
```

Comment:
```suggestion
if [ $# -ne 3 ]
```

cf. https://github.com/YunoHost/yunorunner/commit/470fb02022e3b923519cd5786eac0ce16ec41f16